README.md

```diff
@@ -20,6 +20,18 @@ This will check that the transitive dependencies of your project do not exhibit
 any binary compatibility conflict, assuming that the methods of your `Compile`
 confliction (in `src/main/`) are all called.
 
+You can ignore certain conflicts with regex match:
+
+```scala
+Compile / missinglinkIgnoreConflicts += "com.google.common.base.Enums.*".r
+```                                                                             
+
+If you're certain a library has conflicts which can be safely ignored and want to speedup checking a bit you can exclude the whole of it from analysis:
+
+```scala
+Compile / missinglinkExcludeDependencies ++= List("ch.qos.logback" % "logback-core" % "1.2.3", "ch.qos.logback" % "logback-classic" % "1.2.3")
```

Comment:
The fact that I have to specify a version sounds weird to me. Because of dependency resolution, I might end up with a different version that was is written here. Will it then be excluded or not?

Consider following the strategy of the original Maven plugin, which does not require a version in its exclude list.

Reply:
the version must match, i believe if you end up with different versions you must reevaluate the conflicts and adjust exclusions accordingly